src/test/java/cms/gov/madie/measure/resources/MeasureControllerMvcTest.java

```diff
@@ -64,6 +77,12 @@ public class MeasureControllerMvcTest {
   @Captor private ArgumentCaptor<Measure> measureArgumentCaptor;
 
   private static final String TEST_USER_ID = "test-okta-user-id-123";
+
+  private static final String HARP_ID_HEADER_KEY = "harp-id";
+  private static final String HARP_ID_HEADER_VALUE = "XxYyZz";
+  private static final String LAMBDA_TEST_API_KEY_HEADER = "api-key";
+  private static final String LAMBDA_TEST_API_KEY_HEADER_VALUE = "9202c9fa";
```

Comment:
Is there isn't a lambda associated with this controller, recommend removing the `LAMBDA` prefix.